src/aap_eda/api/serializers/activation.py

```diff
@@ -37,7 +37,11 @@
 from aap_eda.api.serializers.rulebook import RulebookRefSerializer
 from aap_eda.api.vault import encrypt_string
 from aap_eda.core import models, validators
-from aap_eda.core.enums import CredentialType, ProcessParentType
+from aap_eda.core.enums import (
+    CredentialType,
```

Comment:
@hsong-rh This should be removed, there are references to this CredentialType but they are old and should be swapped with DefaultCredentialType